### PR TITLE
add test cucumbers for get all endpoints

### PR DESCRIPTION
**Pull request by edsonarios**

### CODE REVIEW

File: src/main/java/api/ApiManager.java [hunk 1]
```diff
@@ -1,34 +1,40 @@
...
 
+import static configuration.env.CONFIG;
 import static io.restassured.RestAssured.given;
+import static configuration.Authentication.token;
 
 public class ApiManager {
 
     private static RequestSpecification buildRequest(ApiRequest apiRequest) {
-        return given().headers(apiRequest.getHeaders())
+        if (apiRequest.getBaseUri() == null) {
```
Comment (Paola-Xaq) [inline src/main/java/api/ApiManager.java:14]: Why is it necessary this if?

File: src/test/java/rest/salesforce/org/runner/RunTests.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package rest.salesforce.org.runner;
+
+import io.cucumber.testng.AbstractTestNGCucumberTests;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+
+/*@CucumberOptions(
```
Comment (JorgeRCV) [inline src/test/java/rest/salesforce/org/runner/RunTests.java:7]: These lines should be deleted

File: src/test/java/rest/salesforce/org/steps/BuildSteps.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+public class BuildSteps {
+    private Logger LOGGER = Logger.getLogger(getClass());
+    ApiRequest apiRequest;
+
+
+    public BuildSteps(ApiRequest apiRequest) {
+        this.apiRequest = apiRequest;
+    }
+
+    @Given("I build {string} request")
```
Comment (JorgeRCV) [inline src/test/java/rest/salesforce/org/steps/BuildSteps.java:17]: The given sentece could be improved

File: src/test/java/rest/salesforce/org/steps/CaseSteps.java [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    ApiResponse apiResponse;
+    String caseIdCreated;
+
+    public CaseSteps(ApiRequest apiRequest, ApiResponse apiResponse) {
+        this.apiRequest = apiRequest;
+        this.apiResponse = apiResponse;
+    }
+
+    /*
+    @Before
```
Comment (JorgeRCV) [inline src/test/java/rest/salesforce/org/steps/CaseSteps.java:17]: Delete this lines

File: src/test/java/rest/salesforce/org/steps/UpdateSteps.java [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+import configuration.Authentication;
+
+public class UpdateSteps {
+    ApiRequest apiRequest;
+    ApiResponse apiResponse = new ApiResponse();
+    String createdId;
+    public Authentication authentication = new Authentication();
+
+
+/*
```
Comment (JorgeRCV) [inline src/test/java/rest/salesforce/org/steps/UpdateSteps.java:14]: Delete the commented code

**Review by JorgeRCV — APPROVED** [review]

**Comment by JorgeRCV** [comment]
There are missing the checkstyle conventions, but will be fixed on the next PRs

**Pull request merged**